concepticondata/concepticon.tsv

```diff
@@ -3000,3 +3000,25 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3004	FORMER	Time	Coming before in time.	Property	
 3005	FAMILIAR	Emotions and values	Knowing somebody or something well.	Property	
 3006	SPACE (AVAILABLE)	Spatial relations	Available space to place things.	Person/Thing	
+3007	EUCALYPT	Agriculture and vegetation	Tree or wood for plants with capsule fruiting bodies belonging to closely related genera found across Australia.	
+3008	DEATH ADDER	Animals	Venomous snake of genus Acanthopis.	
+3009	ECHIDNA	Animals	Egg-laying mammal of the Tachyglossidae family.	
+3010	WHITE COCKATOO	Animals	Large white cockatoo, species C. galerita.	
+3011	WAGTAIL	Animals	Passerine bird of genus Motacilla.	
+3012	GRUB	Animals	Larva of Coleoptera order.	
+3013	BANDICOOT	Animals	Marsupial of Peramelemorphia order.	
+3014	BLACK COCKATOO	Animals	Black cockatoo, usually the Palm cockatoo.	
+3015	BLACK DUCK	Animals	Pacific black duck (Anas superciliosa).	
+3016	CROSS COUSIN	Kinship	Cousin from a parent's opposite-sex sibling (in relation to parallel cousin).	
+3017	PARALLEL COUSIN	Kinship	Cousin from a parent's same-sex sibling (in relation to cross cousin).	
+3018	FATHER'S FATHER	Kinship	Male grandparent of the father side (in relation to mother's father).	
+3019	MOTHER'S FATHER	Kinship	Male grandparent of the mother side (in relation to father's father).	
+3020	FATHER'S MOTHER	Kinship	Female grandparent of the father side (in relation to mother's mother).	
+3021	MOTHER'S MOTHER	Kinship	Female grandparent of the mother side (in relation to father's mother).	
+3022	FIRE STICK	Basic actions and technology	Sticks used specifically to set fire to smt. or to burn.	
+3023	GOANNA	Animals	Australian monitor lizard of the genus Varanus.	
+3024	QUOLL	Animals	Marsupial of the genus Dasyurus.	
+3025	SAND HILL	The physical world	Ecological community or xeric wildfire-maintained ecosystem (different from sand dune).	
+3026	URINATE	The body	To pass or discharge urine.	
```

Comment:
We have defecate for "to shit", and we have "to pee" as well, I am sure, so you should link to this concept, instead of creating a new one.

Reply:
I will look for it, I did find it really strange for such a basic concept to be missing.

Reply:
Its 592 PISS ;)